tests/core/scene/a-scene.test.js

```diff
@@ -28,6 +28,24 @@ suite('a-scene (without renderer)', function () {
     test('initializes scene object', function () {
       assert.equal(this.el.object3D.type, 'Scene');
     });
+
+    test('does not initialize systems', function (done) {
+      var sceneEl = document.createElement('a-scene');
+      process.nextTick(function () {
```

Comment:
Do you really need `nextTick` here? I believe that createdCallback is called synchronously